.travis.yml

```diff
@@ -62,21 +63,23 @@ matrix:
         - "3.6"
 
       before_install:
-        - "travis_retry pip install --upgrade pip setuptools py pipenv"
-        - "travis_retry pip install twine wheel coveralls"
-
+        - curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+        - source $HOME/.poetry/env
+        - travis_retry poetry install
+        - travis_retry pip install twine wheel coveralls
+        
       install:
         - "./scripts/bootstrap --ci"
```

Comment:
since you are touching all the others, we can remove ``"`` from here as well

Reply:
yes, i missed it.